js/index.js

```diff
@@ -0,0 +1,72 @@
+$(document).ready(function () { 
+
+	// // hide all answers/
+	$('#learnMoreText, #readMoreText, #read-less, #learn-less').addClass('hide');
```

Comment:
It looks like these items already have the `hide` class on them. Also, I don't think there's anything in your page with an id `learn-less`